app/src/main/java/org/oppia/app/utility/Logger.kt

```diff
@@ -0,0 +1,197 @@
+package org.oppia.app.utility
+
+import android.util.Log
+import org.oppia.app.application.OppiaApplication
+
+import java.io.BufferedWriter
+import java.io.File
+import java.io.FileWriter
+import java.io.IOException
+import java.text.SimpleDateFormat
+import java.util.Calendar
+import java.util.Locale
+
+/** Wrapper class for Android Log. */
+object Logger {
```

Comment:
Please make this a class & make it injectable, e.g.:

```class Logger @Inject constructor(@ApplicationContext context: Context)```

Reply:
done